app/views/admin/useremail.html.erb

```diff
@@ -3,7 +3,7 @@
 
   <h3>User lookup by email:</h3> 
 
-  <p>Moderators and admins have the ability to search for usernames associated with a given email address. This ability is meant to assist in answering requests submit by email where the user forgot specify their username; clearly the email address of the sender is known.</p>
+  <p>Moderators and admins have the ability to search for usernames associated with a given email address. This ability is meant to assist in answering requests submitted by email where the user forgets to specify their username; clearly the email address of the sender is known.</p> address of the sender is known.</p>
```

Comment:
Just a slight change and you're ready to go!
```suggestion
  <p>Moderators and admins have the ability to search for usernames associated with a given email address. This ability is meant to assist in answering requests submitted by email where the user forgets to specify their username; clearly the email address of the sender is known.</p>
```
After closeing the ```</p>``` tag, you have added some repeated line. Kindly delete those. 
Rest all is good!